contracts/DelayedStakeBank.sol

```diff
@@ -0,0 +1,45 @@
+//! TODO: Determine dual licensing for this project
+//! since HarbourProject uses GPLv3 and parity contacts use Apache.
+//! https://softwareengineering.stackexchange.com/questions/197710/how-to-use-gpl-v3-with-apache-license-2-0
+
+pragma solidity ^0.4.23;
+
+import "./StakeBank.sol";
+
+
+contract DelayedStakeBank is StakeBank {
+    uint256 unstakeDelay;
+    mapping (address => uint256) lastStaked;
+
+    /// @param _token Token that can be staked.
```

Comment:
why three `///`?

can we stick to the usual

```
/**
 * block of comments
 */
```
style for block comments and
```
// single line comment
```
for single line comments?

Reply:
Yes, I"ll change all the broken comments & documentation on next commit.